index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Biruk Bereka Full-Stack Developer</title>
+    <link rel="stylesheet" href="index.css" />
+  </head>
+  <body>
+    <header class="header">
+      <a href="index.html"><h1 class="my-name">Biruk Bereka</h1></a>
```

Comment:
[OPTIONAL] H1 tags are specifically reserved for main keywords and subject matter or what the overall post is about. Kindly consider replacing it with a span tag or different tag for SEO and semantic reasons :wink: